addons/im_livechat/models/discuss_channel.py

```diff
@@ -680,6 +681,15 @@ def _apply_livechat_feedback(self, rate, reason=None):
             rating_id=rating_sudo.id,
             subtype_xmlid="mail.mt_comment",
         )
+        if rating_sudo.rated_partner_id not in self.channel_member_ids.partner_id:
+            rating_sudo.rated_partner_id.sudo().user_ids._bus_send(
+                "livechat_rating_notification", {
+                    "rating_url": rating_sudo.rating_image_url,
+                    "visitor_name": user.partner_id.name if user.partner_id else None,
```

Comment:
We shouldn't ignore guest name.

Ideally it's best if this is sent as store data with all the necessary fields.